packages/components/src/components/toggle-switch/toggle-switch.jsx

```diff
@@ -1,21 +1,23 @@
-import React from 'react';
+import * as React from 'react';
```

Comment:
```suggestion
import React from 'react';
```

Reply:
Same as other comment (adding here for lurkers):

- https://epicreact.dev/importing-react-through-the-ages/